docs/configuration.asciidoc

```diff
@@ -593,6 +593,26 @@ Additionally, spans that lead to an error can't be discarded.
 |============
 
 
+[float]
+[[config-enable-open-telemetry-bridge]]
+==== `EnableOpenTelemetryBridge` (added[1.13])
+
+Setting this option to true will enable the <<opentelemetry-bridge, OpenTelemetry Bridge>>. Briefly, the OpenTelemetry Bridge allows one to use the vendor-neutral OpenTelemetry Tracing API (which is the https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.activity?view=net-5.0[Activity API] in .NET) to manually instrument your code, and have the Elastic .NET APM agent handle those API calls.
+
+[options="header"]
+|============
+| Environment variable name            | IConfiguration key
+| `ELASTIC_APM_ENABLEOPENTELEMETRYBRIDGE` | `ElasticApm:EnableOpenTelemetryBridge`
```

Comment:
Let's discuss the naming, because I don't think this is ok.

What I documented is the current status - so this is in the agent code and right now users use these values.

Issues I see:
- For the environment variable we should use `_`, so it should be `ELASTIC_APM_ENABLE_OPENTELEMETRY_BRIDGE` (or `OPEN_TELEMETRY`?)
- The Node.js agent uses [`ELASTIC_APM_OPENTELEMETRY_BRIDGE_ENABLED`](https://github.com/elastic/apm-agent-nodejs/blob/main/docs/configuration.asciidoc#opentelemetrybridgeenabled) - if we change this in .NET, I think we should be in-sync with other agents. This config is not part of the spec, because most agents integrate with OTel differently ([see here](https://www.elastic.co/guide/en/apm/agent/java/current/opentelemetry-bridge.html#otel-getting-started)).
- If we change it, that'll be a breaking change for existing users. This feature is not yet GA if I remember correctly, so technically we can do it, but it's still not nice.

What are other's thoughts on this?

I tend to prefer renaming to `ELASTIC_APM_OPENTELEMETRY_BRIDGE_ENABLED` (and adapt the `IConfiguration key` as well), but no strong feelings...

Reply:
Not sure if my vote counts, but I vote for `ELASTIC_APM_OPENTELEMETRY_BRIDGE_ENABLED`. If it's not GA yet, I think we should take advantage of that fact and align on naming. That's what preview/beta is for, right?

Reply:
Also, hi. I'm back from vacation now and taking a look at this PR 👀

Reply:
Of course your vote counts @bmorelli25! Welcome back 👏 

Cool - so far, the plan is to change this into `ELASTIC_APM_OPENTELEMETRY_BRIDGE_ENABLED`. @z1c0 ok with that? If so, I'll update this PR to use the new name and open another one where I change it in the code as well.

Reply:
Discussed in the .NET weekly -> updated the config name to `ELASTIC_APM_OPENTELEMETRY_BRIDGE_ENABLED`.